pkg/lib/cockpit-components-table.jsx

```diff
@@ -247,6 +248,7 @@ ListingTable.propTypes = {
     caption: PropTypes.string,
     emptyCaption: PropTypes.node,
     columns: PropTypes.arrayOf(PropTypes.oneOfType([PropTypes.object, PropTypes.string])),
+    props: PropTypes.object.isRequired,
```

Comment:
I don't understand this commit -- This introduces a new required property, but does not update any of the existing code to add it. Also, "props" is very generic -- everything is a property, and IMHO they should be flat, not indirect like this. The only allowed (and expected) key in it is `key` right now, right?

Reply:
That was a mistake actually - props is a mandatory key inside rows. Sorry for confusion.

Reply:
This only mandatory is the row = > { props: { key: THIS IS MANDATORY} }
See https://github.com/cockpit-project/cockpit/pull/14447/files#diff-817714eadf61479bc760742bcca467a8R46